docs/monitoring/monitoring.md

```diff
@@ -114,6 +114,149 @@ spec.values.prometheus-node-exporter.resources.requests.memory: 30Mi
+
+![](/img/v1.1/monitoring/webhook-receiver-1.png)
+
+Fill in the required parameters and click `Create`.
+
+![](/img/v1.1/monitoring/webhook-receiver-2.png)
+
+#### Configure AlertmanagerConfig from CLI
+
+You can also add `AlertmanagerConfig` from the CLI.
+
+Exampe: a Webhook receiver in the `default` namespace.
+
+```
+cat << EOF > a-single-receiver.yaml
+apiVersion: monitoring.coreos.com/v1alpha1
+kind: AlertmanagerConfig
+metadata:
+  name: amc-example
+  # namespace: your value
+  labels:
+    alertmanagerConfig: example
+spec:
+  route:
+    continue: true
+    groupBy:
+    - cluster
+    - alertname
+    receiver: "amc-webhook-receiver"
+  receivers:
+  - name: "amc-webhook-receiver"
+    webhookConfigs:
+    - sendResolved: true
+      url: "http://192.168.122.159:8090/"
+EOF
+
+# kubectl apply -f a-single-receiver.yaml
+alertmanagerconfig.monitoring.coreos.com/amc-example created
+
+# kubectl get alertmanagerconfig -A
+NAMESPACE   NAME          AGE
+default     amc-example   27s
+
+```
+
+#### Example of an Alert Received by Webhook
+
+Alerts sent to the webhook server will be in the following format:
+
+```
+{
+'receiver': 'longhorn-system-amc-example-amc-webhook-receiver',
+'status': 'firing', 
+'alerts': [], 
+'groupLabels': {}, 
+'commonLabels': {'alertname': 'LonghornVolumeStatusWarning', 'container': 'longhorn-manager', 'endpoint': 'manager', 'instance': '10.52.0.83:9500', 'issue': 'Longhorn volume is Degraded.',
+'job': 'longhorn-backend', 'namespace': 'longhorn-system', 'node': 'harv2', 'pod': 'longhorn-manager-r5bgm', 'prometheus': 'cattle-monitoring-system/rancher-monitoring-prometheus',
+'service': 'longhorn-backend', 'severity': 'warning'},
+'commonAnnotations': {'description': 'Longhorn volume is Degraded for more than 5 minutes.', 'runbook_url': 'https://longhorn.io/docs/1.3.0/monitoring/metrics/',
+'summary': 'Longhorn volume is Degraded'},
+'externalURL': 'https://192.168.122.200/api/v1/namespaces/cattle-monitoring-system/services/http:rancher-monitoring-alertmanager:9093/proxy',
+'version': '4',
+'groupKey': '{}/{namespace="longhorn-system"}:{}',
+'truncatedAlerts': 0
+}
+```
+
+:::note
+
+Different receivers may present the alerts in different formats. For details, please refer to the related documents.
+
+:::
+
+#### Known Limitation
+
+The `AlertmanagerConfig` is enforced by the `namespace`. Gloabl-level `AlertmanagerConfig` without a namespace is not supported.
+
+We have already created a [GithHb issue](https://github.com/harvester/harvester/issues/2760) to track upstream changes. Once the feature is available, `Harvester` will adopt it.
+
+### View and Manage Alerts
+
+#### From Alertmanager Dashboard
+
+You can visit the original dashboard of `Alertmanager` from below link.
+
+> https://the-cluster-vip/api/v1/namespaces/cattle-monitoring-system/services/http:rancher-monitoring-alertmanager:9093/proxy/#/alerts
```

Comment:
```suggestion
You can visit the original dashboard of `Alertmanager` from the link below. Note that you need to replace <cluster-vip> with the actual cluster-vip:

> https://<cluster-vip>/api/v1/namespaces/cattle-monitoring-system/services/http:rancher-monitoring-alertmanager:9093/proxy/#/alerts
```

Reply:
Thanks for explaining the links! If so, I think it's better to tell users the need to replace the variable.

Reply:
seems `<cluster-vip>` is not fit here.

![image](https://user-images.githubusercontent.com/31133476/197510628-83224651-0203-47e4-be3a-cf71ef1075bb.png)

Reply:
Then maybe we should remove the angle brackets ^^

Reply:
keep `the-cluster-vip` and without `<>` makes CI happy.